code/game/machinery/cryo.dm

```diff
@@ -115,6 +115,7 @@
 			visible_message("[user] climbs into the cryo cell.")
 		else
 			visible_message("[user] puts [L.name] into the cryo cell.")
+			log_game("[key_name(user)] puts [key_name(L.name)] into a cryo cell at [COORD(src)].")

```

Comment:
"user *put* key_name" (Past-tense)

Reply:
Also don't use `log_game`, that's not the right logging proc to use
Use `add_attack_logs` instead, which will actually be visible to admins, and leave logs that can be inspected on each mob later.